lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -327,7 +327,7 @@ private Hover collectProperty(IPositionRequest request, String property) {
 			for (Entry<String, String> prop : allProps.entrySet()) {
 				String mavenProperty = prop.getKey();
 				if (property.equals(mavenProperty)) {
-					return new Hover(new MarkupContent("plaintext", "Property: " + mavenProperty
+					return new Hover(new MarkupContent("markdown", "Property: " + mavenProperty
```

Comment:
You should take care if client can support markdown.

To do that:

`boolean markdown = request.canSupportMarkupKind(MarkupKind.MARKDOWN);`

After that I suggest you that you do like we have done for entites doc (for hover and completion). See https://github.com/eclipse/lemminx/blob/374ea32776f24a910f6a2541d0f66dd21cff1266/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/entities/EntitiesDocumentationUtils.java#L91

Reply:
At the moment, I think we can just not care too much about such clients as they should be marginal. We'll work on that when we have a request for it from some integrator and we consider it's high priority. Realistically, it's more the kind of things we will let the people who want it implement.

Reply:
If you use https://github.com/eclipse/lemminx/blob/374ea32776f24a910f6a2541d0f66dd21cff1266/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/MarkupContentFactory.java#L41 everything should be supported